contracts/Strategy.sol

```diff
@@ -105,6 +158,119 @@ contract Strategy is BaseStrategy {
             uint256(int256(supply) + delta);
         uint256 newSupplyRate = cToken.getSupplyRate(newUtilization) *
             SECONDS_PER_YEAR;
-        return newSupplyRate;
+
+        uint rewardRate = getRewardAprForSupplyBase(getPriceFeedAddress(comp), delta);
+
+        return newSupplyRate + rewardRate;
+    }
+
+    function _tend() internal override {
+        // claim rewards, sell rewards, reinvest rewards
+        _claimCometRewards();
+        if(tradeFactory == address(0)) {
```

Comment:
This check is also [here](https://github.com/jmonteer/yearnv3-strat-compoundv3-lender/pull/4/files#diff-3e804d467472e49fe8afea1899d96f26655f3ecd641555a801abcea4f12af598R182) so one could be removed